tests/subsys/fs/fs_api/src/test_multi_fs.c

```diff
@@ -105,12 +115,39 @@ static int test_fs_external(void)
 /**
  * @brief Multi file systems register and unregister
  *
- * @details register and unregister two file systems to test
- *          the system support multiple file system simultanously
+ * @details
+ * Test Objective:
+ * - register and unregister two file systems to test the system support
+ *   multiple file system simultanously
+ *
+ * Testing techniques:
+ * - Interface testing
+ * - function and block box testing
+ * - dynamic analysis and testing
+ *
+ * Prerequisite Conditions:
+ * - N/A
+ *
+ * Input Specifications:
+ * - N/A
+ *
+ * Test Procedure:
+ * -# regiter two filesystems and mount them
+ * -# check how many filesystems have been rigistered via fs_readmount()
```

Comment:
`fs_readmount` does not iterate through the list of registered file systems but mounted file systems.